python/lsst/ts/m2/csc.py

```diff
@@ -684,27 +689,32 @@ def _assert_enabled_csc_and_controller(
         self.controller_cell.assert_controller_state(message_name, allowed_curr_states)
         self.assert_enabled()
 
-    async def _write_command_to_server(
-        self, message_name: str, timeout: float, message_details: dict | None = None
+    async def _execute_command_to_server(
```

Comment:
I think I liked the previous name better, this new name reads a bit weird. Maybe remove "_to_server" and just call it `_execute_command`?

Reply:
Renamed in the commit: [71466d9](https://github.com/lsst-ts/ts_m2/pull/45/commits/71466d9802e0a0c6b2ae58e37845e0cd3430fc94). Thanks!